ui/apps/everest/src/components/cluster-form/resources/constants.ts

```diff
@@ -68,14 +68,16 @@ export const DEFAULT_SIZES = {
   },
 };
 
-export const SHARDING_DEFAULTS = {
-  [DbWizardFormFields.shardConfigServers]: {
-    min: '1',
-    max: '7',
-  },
-  [DbWizardFormFields.shardNr]: {
-    min: '1',
-  },
+export const DEFAULT_CONFIG_SERVERS = ['1', '3', '5', '7'];
+
+export const MIN_NUMBER_OF_SHARDS = '1';
+
+export const getDefaultNumberOfconfigServersByNumberOfNodes = (
+  numberOfNodes: number
+) => {
+  if (DEFAULT_CONFIG_SERVERS.includes(numberOfNodes.toString())) {
+    return numberOfNodes <= 5 ? numberOfNodes.toString() : '7';
```

Comment:
not sure I understand this.

`DEFAULT_CONFIG_SERVERS.includes(numberOfNodes.toString())` means `numberOfNodes` has to be 1, 3, 5 or 7.
If `numberOfNodes > 5`, we can still return `numberOfNodes.toString()`, no need for ternary operator

Reply:
@fabio-silva we have only 4 values for config servers, that's why
![image](https://github.com/user-attachments/assets/a2e0e034-0425-4c45-a550-918c339f0dc4)

Reply:
Right, but here's the thing:
- Options are 1, 3, 5, 7
- `numberOfNodes <= 5 ? numberOfNodes.toString() : '7';` -> if number is greater than 5, it can only be 7 anyways, so we can just do `numberOfNodes.toString() : '7'` right away without this condition

Reply:
fixed in https://github.com/percona/everest/pull/746/commits/a7a359f836db0456bfadd1842b1af6e6f877a4a8